src/v1/plugins/github/render.js

```diff
@@ -81,6 +81,13 @@ function num(x: IssueReference | PullRequestReference) {
 function authors(authorable: {+authors: () => AuthorReference[]}) {
   // TODO: modify to accomodate multi-authorship
   const authorRefs = authorable.authors();
-  const firstAuthor = authorRefs[0].get();
-  return firstAuthor != null ? firstAuthor.login() : "[unknown]";
+  const firstAuthorRef = authorRefs[0];
+  if (firstAuthorRef == null) {
```

Comment:
To check my understanding: the problem is that `authorRefs` is an empty
array, right? not an array containing a null element? (If the latter
were the case, then that would mean that our `.authors()` function has
an unsound type signature.)

Reply:
Correct.